package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   name: "cultofcoders:redis-oplog",
-  version: "3.0.0-rc.3",
+  version: "3.1.0",
```

Comment:
This should be 3.0.0-rc.3
There was no 3.0 release. If this is enough to get us Meteor 3 compat I'm happy to release this as 3.0

Reply:
All tests pass and some companies are already using it in production. I think it should be 3.0 do reduce confusion IMO.